packages/consumption/test/modules/requests/IncomingRequestsController.test.ts

```diff
@@ -384,7 +384,7 @@ describe("IncomingRequestsController", function () {
             expect(validationResult.items[1].items[2].isError()).toBe(true);
         });
 
-        test("throws on terminated relationship", async function () {
+        test("returns 'error' on terminated relationship", async function () {
```

Comment:
now this sounds like the method returns a string 'error' ...

Reply:
I'm just using the style of this file